docs/sources/configuration/_index.md

```diff
@@ -186,7 +186,7 @@ The `server_config` block configures the HTTP and gRPC server of the launched se
 [http_listen_port: <int> | default = 80]
 
 # gRPC server listen host
-# CLI flag: -server.grpc-listen-address
+# CLI flag: -server.comlisten-address
```

Comment:
Changed by mistake? I think the flag has not changed.

Reply:
ho boy.

Reply:
done @sandeepsukhani thanks !